### PR TITLE
Feature/ab#744 dev cluster

### PR DESCRIPTION
**Pull request by witchpou**
Add dev cluster to helm repository in order to make it easier for developers to create a new cluster.

### CODE REVIEW

File: helmfiles/cityos/applyHelmfile.sh [hunk 1]
```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
-
-set -a; . .env; set +a; helmfile apply
\ No newline at end of file
+# set hostname in lower case
+export HOSTNAME=${HOSTNAME,,}
```
Comment (DerCed) [inline helmfiles/cityos/applyHelmfile.sh:3]: What are those `,` for?

File: k3s-dev-cluster/installPrerequisites.sh [hunk 2]
```diff
@@ -1,22 +1,18 @@
 #!/bin/bash
 if ! [ -x "$(command -v helmfile)" ]; then
+    echo "installing plugin helmfile ..."
     mkdir tmp
     cd tmp
-    wget https://github.com/helmfile/helmfile/releases/download/v0.153.1/helmfile_0.153.1_linux_amd64.tar.gz
-    tar -xzvf helmfile_0.153.1_linux_amd64.tar.gz
+    wget https://github.com/helmfile/helmfile/releases/download/v0.154.0/helmfile_0.154.0_linux_amd64.tar.gz
```
Comment (DerCed) [inline k3s-dev-cluster/installPrerequisites.sh:6]: Is there a latest tag to use?

**Review by DerCed — APPROVED** [review]
Looks good to me :)

File: k3s-dev-cluster/installPrerequisites.sh (hunk 2 above)
Comment (witchpou) [reply k3s-dev-cluster/installPrerequisites.sh:6]: yes

**Pull request merged**